### DISCUSSION TITLE
React.js Page is broken on vercel.

### DISCUSSION THREAD

**Discussion by 1nsane-dev** [vercel/community#2077] [Help]
### Summary

I built a project using create-react-app and deployed on vercel.
The pages like `/user/edit`, `/user/new` are broken, although the pages like `/user`, `/home` works fine.
Could you support me to figure out the issue?

**Comment by amyegan** [comment]
Hi @1nsane-dev. If you can share your repo or a [minimal reproduction](https://vercel.com/support/articles/creating-a-minimal-reproducible-example), I can take a look to help you debug this
